azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/metrics/AzureMonitorPropertiesConfigAdapter.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.autoconfigure.metrics;
+
+import io.micrometer.azuremonitor.AzureMonitorConfig;
+import org.springframework.boot.actuate.autoconfigure.metrics.export.properties.StepRegistryPropertiesConfigAdapter;
+
+/**
+ * Adapter to convert {@link AzureMonitorProperties} to a {@link AzureMonitorConfig}.
+ *
+ * @author Dhaval Doshi
+ */
+public class AzureMonitorPropertiesConfigAdapter extends StepRegistryPropertiesConfigAdapter<AzureMonitorProperties>
+    implements AzureMonitorConfig {
+
+  AzureMonitorPropertiesConfigAdapter(
+      AzureMonitorProperties properties) {
```

Comment:
Newline is not necessary here? The default modifier is on purpose?

Reply:
Yes default modifier is for purpose. This constructor is not used publicaly.